### PR TITLE
Update documentation for orka-full

### PR DESCRIPTION
**Pull request by ikoevska**
The documentation for `orka-full` is now updated:

* Documented manual install
* Added example for the configMap
* Separated script install/config in a separate file
* Added information about how to download the scripts

### CODE REVIEW

File: task/orka-full/0.1/SCRIPTS.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Install & Configure with Scripts
+
+For easier install and configuration of the Orka Tekton `Tasks`, you can use the provided scripts.
+
+* [TL;DR;](#tldr)
+* [How to install the task](#how-to-instal-the-task)
```
Comment (spikeburton) [inline task/orka-full/0.1/SCRIPTS.md]:
```suggestion
* [How to install the task](#how-to-install-the-task)
```

File: task/orka-full/0.1/SCRIPTS.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Install & Configure with Scripts
+
+For easier install and configuration of the Orka Tekton `Tasks`, you can use the provided scripts.
+
+* [TL;DR;](#tldr)
```
Comment (spikeburton) [inline task/orka-full/0.1/SCRIPTS.md]:
```suggestion
* [TL;DR](#tldr)
```

File: task/orka-full/0.1/SCRIPTS.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+* [How to install the task](#how-to-instal-the-task)
+* [How to store your credentials](#how-to-store-your-credentials)
+
+## TL;DR;
```
Comment (spikeburton) [inline task/orka-full/0.1/SCRIPTS.md]:
```suggestion
## TL;DR
```

File: task/orka-full/0.1/README.md [hunk 4]
```diff
@@ -28,33 +26,31 @@ See also: [GCP-MacStadium Site-to-Site VPN](https://docs.macstadium.com/docs/goo
...
 
-**Custom namespace installation**
+Omit `--namespace` if installing in the `default` namespace.
 
-To install in a custom namespace, run the following command against your preferred namespace and your actual Orka API endpoint:
+You can use the following sample `orka-configuration.yaml`. Make sure to provide the correct Orka API endpoint for your Orka environment.
 
-```sh
-NAMESPACE=tekton-orka ORKA_API=http://10.221.188.100 ./install.sh --apply
+```yaml
```
Comment (spikeburton) [inline task/orka-full/0.1/README.md:31]: Thought - we might want to put the YAML example before the `kubectl apply` command - at first glance I was a bit confused as to where/what `orka-configuration.yaml` is, and other folks might be confused at first as well. Same below with the credentials.

File: task/orka-full/0.1/SCRIPTS.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Omit `NAMESPACE` if installing in the `default` Kubernetes namespace.
+
+#### Install the task
```
Comment (spikeburton) [inline task/orka-full/0.1/SCRIPTS.md:13]: `nit`: suggest changing these to size 3 headings here, e.g. `### Install the task`

File: task/orka-full/0.1/README.md [hunk 6]
```diff
@@ -122,7 +91,17 @@ stringData:
   password: admin
 ```
 
-#### Using an SSH key
+> **TIP:** Did you know you could use a script for easier setup?
+>
+> curl -LO https://raw.githubusercontent.com/tektoncd/catalog/master/task/orka-full/0.1/add-orka-creds.sh && chmod 755 add-orka-creds.sh
+> NAMESPACE=\<namespace\> EMAIL=\<email\> PASSWORD=\<password\> ./add-orka-creds.sh --apply
```
Comment (spikeburton) [inline task/orka-full/0.1/README.md]: The formatting of this looks a bit off to me when rendered. Maybe we consider putting these commands inside of a script block, i.e. with triple backticks, or just having the link to SCRIPTS.md

**Review by spikeburton — COMMENTED** [review]
Awesome, I think that having the steps for running the scripts in a separate doc is a great approach.

Couple of things:
1. let's remove all the `.DS_Store` files - we can add a `.gitignore` if needed
1. we should probably add absolute links for the samples as well - I think these will end up looking something like `https://github.com/tektoncd/catalog/blob/master/task/orka-full/0.1/samples/use-ssh-key.yaml`

File: task/orka-full/0.1/SCRIPTS.md (hunk 5 above)
Comment (ikoevska) [reply task/orka-full/0.1/SCRIPTS.md:13]: The H4 is deliberate. H3 just doesn't look good (it's too big and becomes a bit redundant with the more detailed section of the same name below). I was looking for a way to implement sectioning but still retain anchors. Otherwise, I would have gone with plain bold formatting.

Let me know what you think.

File: task/orka-full/0.1/README.md (hunk 6 above)
Comment (ikoevska) [reply task/orka-full/0.1/README.md]: Script blocks don't work within the block quote element. Simple code formatting doesn't look good either. Let's just drop these and link to the TL;DR sections instead.

**Comment by ikoevska** [comment]
@spikeburton I addressed pretty much everything, except the `nit` comment. Please, check my response and let me know what you think :) 

Ideally, there would be a `.gitignore` because I have to manually clean up the `.DS_Store` stuff every time. I'm just not sure it needs to be added with this PR.

File: task/orka-full/0.1/SCRIPTS.md (hunk 5 above)
Comment (spikeburton) [reply task/orka-full/0.1/SCRIPTS.md:13]: Makes sense :)

**Review by spikeburton — APPROVED** [review]
LGTM
